forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -171,17 +173,15 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Captures a Forage Payment associated with an EBT card
-     *
-     * @param params The parameters required for payment capture, including
-     * reference to a ForagePINEditText and a Payment ref
+     * Captures a payment via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element.
      *
-     * @return A ForageAPIResponse indicating the success or failure of the
-     * payment capture. On success, returns a confirmation of the transaction.
-     * On failure, provides a detailed error response.
-     *
-     * @throws ForageConfigNotSetException If the passed ForagePANEditText instance
-     * hasn't had its ForageConfig set via .setForageConfig().
+     * @param CapturePaymentParams A model that passes a [`foragePinEditText`][com.joinforage.forage.android.ui.ForagePINEditText] instance and a `paymentRef`, returned by the [Create a Payment](https://docs.joinforage.app/reference/create-a-payment) endpoint, as the [CapturePaymentParams] that Forage uses to capture a payment. 
+     * @return A [ForageApiResponse] object. 
+     * 
+     * On success, the object confirms the transaction. 
```

Comment:
we should mention that the response includes a Forage `Payment` object and link the [Create a Payment page](https://docs.joinforage.app/reference/create-a-payment)

I'll chat with FX at our office hours to see if we can improve our response types – as I believe it is currently the client's responsibility to define their own Kotlin types and deserialize the `ForageApiResponse<String>` responses

@devinmorgan @dleis612 to correct me if I'm wrong